apps/site/src/pages/[shortname]/blocks/[block-slug].page.tsx

```diff
@@ -301,10 +302,46 @@ const BlockPage: NextPage<BlockPageProps> = ({
                 />
               )}
               <Typography
-                sx={{ display: { xs: "flex", md: "unset" } }}
                 variant="bpHeading1"
+                sx={{
+                  display: "flex",
+                  flexWrap: "wrap",
+                  alignItems: "center",
+                  columnGap: 2.5,
+                  rowGap: 0.5,
+                }}
               >
-                {blockMetadata.displayName}
+                <Box component="span">{blockMetadata.displayName} </Box>
+
+                {blockMetadata.verified ? (
+                  <Box
+                    sx={{
+                      display: "inline-flex",
+                      alignItems: "center",
+                      gap: 0.5,
+                      borderRadius: 7.5,
+                      background: "#FBF7FF",
```

Comment:
We should look at naming this colour at some point as its being used more and more